### DISCUSSION TITLE
ClickHouse 社群指标模型 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#32] [Comments]
# ClickHouse 社群指标模型 | 夜天之书

数字化时代下绝大多数工作都有关键绩效指标（KPI）指导，任何组织都期望找到一个合理的指标来校准战略方向，衡量工作成果。 然而，并非所有工作都能有 KPI 准确地衡量产出。著名软件工程师，同时也是《重构》《分析模式》等书籍的作者 Martin Fowler 曾经写过一篇博客论证软件工程师的生产力是无法衡量的。 开源社群成员的重要组成部分，也是一切价值的核心飞轮就是这样一群软件工程师。这是否意味着开源

https://www.tisonkun.org/2022/07/25/clickhouse-metrics-model/

**Comment by ZoomQuiet** [comment]
#是也乎

WoW 这比 CHOSS 一堆理论要来的直接的多, 
直接给了一个可运行的模型
